AK/Debug.h.in

```diff
@@ -7,537 +7,537 @@
 #endif
 
 #ifndef SQLSERVER_DEBUG
-#cmakedefine01 SQLSERVER_DEBUG
+#    cmakedefine01 SQLSERVER_DEBUG
 #endif
 
 #ifndef SYNTAX_HIGHLIGHTING_DEBUG
-#cmakedefine01 SYNTAX_HIGHLIGHTING_DEBUG
+#    cmakedefine01 SYNTAX_HIGHLIGHTING_DEBUG
 #endif
 
 #ifndef SYSCALL_1_DEBUG
-#cmakedefine01 SYSCALL_1_DEBUG
+#    cmakedefine01 SYSCALL_1_DEBUG
 #endif
 
 #ifndef SYSTEM_MENU_DEBUG
-#cmakedefine01 SYSTEM_MENU_DEBUG
+#    cmakedefine01 SYSTEM_MENU_DEBUG
 #endif
 
 #ifndef SYSTEMSERVER_DEBUG
-#cmakedefine01 SYSTEMSERVER_DEBUG
+#    cmakedefine01 SYSTEMSERVER_DEBUG
 #endif
 
 #ifndef TERMCAP_DEBUG
-#cmakedefine01 TERMCAP_DEBUG
+#    cmakedefine01 TERMCAP_DEBUG
 #endif
 
 #ifndef TERMINAL_DEBUG
-#cmakedefine01 TERMINAL_DEBUG
+#    cmakedefine01 TERMINAL_DEBUG
 #endif
 
 #ifndef TEXTEDITOR_DEBUG
-#cmakedefine01 TEXTEDITOR_DEBUG
+#    cmakedefine01 TEXTEDITOR_DEBUG
 #endif
 
 #ifndef TIME_ZONE_DEBUG
-#cmakedefine01 TIME_ZONE_DEBUG
+#    cmakedefine01 TIME_ZONE_DEBUG
 #endif
 
 #ifndef TLS_DEBUG
-#cmakedefine01 TLS_DEBUG
+#    cmakedefine01 TLS_DEBUG
 #endif
 
 #ifndef TLS_SSL_KEYLOG_DEBUG
-#cmakedefine01 TLS_SSL_KEYLOG_DEBUG
+#    cmakedefine01 TLS_SSL_KEYLOG_DEBUG
 #endif
 
 #ifndef TOKENIZER_TRACE_DEBUG
-#cmakedefine01 TOKENIZER_TRACE_DEBUG
+#    cmakedefine01 TOKENIZER_TRACE_DEBUG
 #endif
 
 #ifndef UCI_DEBUG
-#cmakedefine01 UCI_DEBUG
+#    cmakedefine01 UCI_DEBUG
 #endif
 
 #ifndef UPDATE_COALESCING_DEBUG
-#cmakedefine01 UPDATE_COALESCING_DEBUG
+#    cmakedefine01 UPDATE_COALESCING_DEBUG
 #endif
 
 #ifndef URL_PARSER_DEBUG
-#cmakedefine01 URL_PARSER_DEBUG
+#    cmakedefine01 URL_PARSER_DEBUG
 #endif
 
 #ifndef UTF8_DEBUG
-#cmakedefine01 UTF8_DEBUG
+#    cmakedefine01 UTF8_DEBUG
 #endif
 
 #ifndef WASM_BINPARSER_DEBUG
-#cmakedefine01 WASM_BINPARSER_DEBUG
+#    cmakedefine01 WASM_BINPARSER_DEBUG
 #endif
 
 #ifndef WASM_TRACE_DEBUG
-#cmakedefine01 WASM_TRACE_DEBUG
+#    cmakedefine01 WASM_TRACE_DEBUG
 #endif
 
 #ifndef WASM_VALIDATOR_DEBUG
-#cmakedefine01 WASM_VALIDATOR_DEBUG
+#    cmakedefine01 WASM_VALIDATOR_DEBUG
 #endif
 
 #ifndef WEBDRIVER_DEBUG
-#cmakedefine01 WEBDRIVER_DEBUG
+#    cmakedefine01 WEBDRIVER_DEBUG
 #endif
 
 #ifndef WEBGL_CONTEXT_DEBUG
-#cmakedefine01 WEBGL_CONTEXT_DEBUG
+#    cmakedefine01 WEBGL_CONTEXT_DEBUG
 #endif
 
 #ifndef WEBSERVER_DEBUG
-#cmakedefine01 WEBSERVER_DEBUG
+#    cmakedefine01 WEBSERVER_DEBUG
 #endif
 
 #ifndef WEB_FETCH_DEBUG
-#cmakedefine01 WEB_FETCH_DEBUG
+#    cmakedefine01 WEB_FETCH_DEBUG
 #endif
 
 #ifndef WEB_WORKER_DEBUG
-#cmakedefine01 WEB_WORKER_DEBUG
+#    cmakedefine01 WEB_WORKER_DEBUG
 #endif
 
 #ifndef WEBP_DEBUG
-#cmakedefine01 WEBP_DEBUG
+#    cmakedefine01 WEBP_DEBUG
 #endif
 
 #ifndef WINDOWMANAGER_DEBUG
-#cmakedefine01 WINDOWMANAGER_DEBUG
+#    cmakedefine01 WINDOWMANAGER_DEBUG
 #endif
 
 #ifndef WSMESSAGELOOP_DEBUG
-#cmakedefine01 WSMESSAGELOOP_DEBUG
+#    cmakedefine01 WSMESSAGELOOP_DEBUG
 #endif
 
 #ifndef WSSCREEN_DEBUG
-#cmakedefine01 WSSCREEN_DEBUG
+#    cmakedefine01 WSSCREEN_DEBUG
```

Comment:
Looks like this is only valid since CMake 3.10. If I have no other comments, would be nice to note that in the comment message.

Reply:
Ah I didn't realise. CLion kept doing it automatically so I gave in. 😅

Reply:
Yeah I mean we're on 3.16 for Lagom so it's a good change